src/mudlet-lua/lua/generic-mapper/generic_mapper.xml

```diff
@@ -1275,7 +1276,7 @@ map.defaults = {
     lang_dirs = {n = 'n', ne = 'ne', nw = 'nw', e = 'e', w = 'w', s = 's', se = 'se', sw = 'sw',
```

Comment:
`lang_dirs` doesn't really fit anymore. Maybe we can adapt the name as well?

Reply:
The name is used in configs.lua that are user personalization and configuration. I think that needs too many changes for a single command that is however related to direction handling.

Reply:
it breaks backward compatibility, you are right. Still makes me sad :cry: